descriptor.md

```diff
@@ -19,7 +19,7 @@ The following fields contain the primary properties that constitute a Descriptor
 
 - **`mediaType`** *string*
 
-  This REQUIRED property contains the media type of the referenced content.
+  This OPTIONAL property contains the media type of the referenced content.
```

Comment:
This should be REQUIRED.  Without it, a consumer will not know how to interpret the referenced content.